File: ios/Theme/CaptureScreen.swift
public class CaptureScreen: ViewStyle {
  private static let KEY: String = "captureScreen"
  private let target: NSDictionary?
  private let font: Font
  private let color: Color

  init(target: NSDictionary?) {
    self.target = Style().getTarget(target, key: CaptureScreen.KEY)
    self.font = Font()
    self.color = Color()

    super.init(target: target, key: CaptureScreen.KEY)
  }
  
  override public func getCornerRadius() -> Int32 {
    return super.getCornerRadius(-1)
  }

  public func getFocusTextColor() -> UIColor {
    return self.color.getColor(self.target, key: "focusTextColor")
  }
  
  public func getFont() -> UIFont {
    return self.font.getTypography(theme: self.target, key: "font")
  }
}

File: android/src/main/java/com/azifacemobile/theme/CaptureScreen.java
package com.azifacemobile.theme;

import android.graphics.Typeface;

import com.azifacemobile.theme.abstracts.ViewStyle;
import com.azifacemobile.utils.Theme;
import com.facebook.react.bridge.ReactApplicationContext;

import org.json.JSONObject;

import javax.annotation.Nullable;

public class CaptureScreen extends ViewStyle {
  private static final String KEY = "captureScreen";
  private final JSONObject target;
  private final Font font;
  private final Color color;

  public CaptureScreen(ReactApplicationContext context, JSONObject target) {
    super(target, KEY);

    this.target = new Theme().getTarget(target, KEY);
    this.font = new Font(context);
    this.color = new Color();
  }

  public int getFrameStrokeColor() {
    return this.color.getColor(this.target, "frameStrokeColor");
  }

  @Nullable
  public Typeface getFont() {
    return this.font.getTypography(this.target, "font");
  }
}
